src/content.js

```diff
@@ -116,8 +143,19 @@ module.exports = function (req, res) {
     async.apply(layout, presented)
   ], function (err, result) {
     if (err) {
-      logger.error("Assembling: " + err);
-      res.status(500).send(page500);
+      var code = 404;
+      if(err.type == "NotFound") {
```

Comment:
Oh, and I'm not certain, but the `err` that we get from a requests call might already include the status code. We could probably check against it directly rather than adding a custom attribute.

Disregard if you already looked for that exact thing and it doesn't exist :wink: